src/test/java/com/fasterxml/jackson/databind/deser/enums/EnumAltIdTest.java

```diff
@@ -213,4 +224,38 @@ public void testEnumWithAliasAndDefaultForUnknownValueEnabled() throws Exception
         MyEnum2352_3 multipleAliases2 = reader.readValue(q("multipleAliases2"));
         assertEquals(MyEnum2352_3.C, multipleAliases2);
     }
+
+    public void testEnumWithDefaultForUnknownValueEnabled() throws JsonProcessingException {
+        final String JSON = a2q("{'value':'ok'}");
+
+        DefaultEnumBean pojo = READER_DEFAULT.forType(DefaultEnumBean.class)
+          .readValue(JSON);
+        assertEquals(MyEnum2352_3.B, pojo.value);
+        // including disabling acceptance
+        try {
+            READER_DEFAULT.forType(StrictCaseBean.class)
+              .readValue(JSON);
+            fail("Should not pass");
+        } catch (InvalidFormatException e) {
+            verifyException(e, "not one of the values accepted for Enum class");
+            verifyException(e, "[JACKSON, OK, RULES]");
+        }
+    }
+
```

Comment:
In addition to testing single `Enum` property it'd be good to see if things work for `EnumSet`s?

I don't remember if things might just work due to delegation, but it might be necessary to change `EnumSetDeserializer`. Adding a test would clarify this.

Reply:
Added the test cases for EnumSet. Please have look if behaviour is expected.Thanks.